### PR TITLE
[Platforms] Accept platforms paper as v1

### LINKED ISSUE #316: [Platforms] Final review and feedback
We've reached out in a few forums for final review and feedback on our [platforms whitepaper](https://github.com/cncf/tag-app-delivery/blob/platforms-v1alpha1/platforms-whitepaper/v1alpha1/paper.md) in advance of publication in April 2023. This issue is to give folks a place to provide general feedback and thoughts. In addition, please feel free to open separate PRs or issues in this repo for significant items. Thank you!

### PR DESCRIPTION
**Pull request by joshgav**
This PR moves the platforms paper from `v1alpha1` to `v1` and merges it into the `main` branch. Merging of this PR will reflect final acceptance of the content for v1.

Fixes #316

### CODE REVIEW

**Comment by joshgav** [comment]
Before merging we must update the image from <https://docs.google.com/drawings/d/1kRr2dCc7Hk1-c_La3kKAgGr_irVPpU3vrVyIZmBmMII/>

**Comment by krumware** [comment]
@joshgav note here referencing the PR: https://deploy-preview-333--tag-app-delivery.netlify.app/whitepapers/platforms/

**Review by AlexsJones — APPROVED** [review]

**Comment by joshgav** [comment]
Image has been updated, and an edit included to address #334.

File: platforms-whitepaper/v1/paper.md [hunk 1]
```diff
@@ -0,0 +1,543 @@
+## Introduction
+
+Inspired by the cross-functional cooperation promised by DevOps, platform
```
Comment (AlexsJones) [inline platforms-whitepaper/v1/paper.md:3]:
```suggestion
Inspired by the cross-functional cooperation promised by DevOps, platform engineering has begun to emerge in enterprises as an explicit form of that cooperation. Platforms curate and present foundational capabilities, frameworks and experiences to facilitate and accelerate the work of internal customers such as application developers, data scientists and information workers. Particularly in cloud computing, platforms have helped enterprises realize values long promised by the cloud, like fast product releases, portability across infrastructures, more secure and resilient products, and greater developer productivity.

This paper intends to support enterprise leaders, enterprise architects and platform team leaders to advocate for, investigate and plan internal platforms for cloud computing. We believe platforms significantly impact enterprises' actual value streams, but only indirectly, so leadership consensus and support are vital to the long-term sustainability and success of platform teams. In this paper, we'll enable that support by discussing what the value of platforms is, how to measure that value, and how to implement platform teams that maximize it.
```

Grammer

**Review by hongchaodeng — APPROVED** [review]

File: platforms-whitepaper/v1/paper.md (hunk 1 above)
Comment (joshgav) [reply platforms-whitepaper/v1/paper.md:3]: Fixed up, thanks!

**Review by roberthstrand — APPROVED** [review]
L(_really_)GTM!

**Pull request merged**